app/uk/gov/hmrc/tariffclassificationfrontend/controllers/SearchController.scala

```diff
@@ -56,4 +57,12 @@ class SearchController @Inject()(authenticatedAction: AuthenticatedAction,
     }
   }
 
+  private def applyDefaultFilters(search: Search): Search = {
```

Comment:
Defaulting to live decisions only is logic specific to the "Advanced Search" view, so I stuck the logic here so we can potentially re-use Search in other places